includes/watchers/class-algolia-post-changes-watcher.php

```diff
@@ -47,6 +47,8 @@ class Algolia_Post_Changes_Watcher implements Algolia_Changes_Watcher {
 	 */
 	private $posts_updated = [];
 
+	public static $pmxi_is_fast_mode;
```

Comment:
@tw2113 We need to add a comment for class member variable here.

![image](https://github.com/WebDevStudios/wp-search-with-algolia/assets/8023941/fb779f25-8432-43bc-b534-96762743326e)